src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -125,98 +115,123 @@ public EventValueInfo(final Class<E> event, final Class<T> c, final Getter<T, E>
 	 * It is recommended to use {@link EventValues#getEventValueGetter(Class, Class, int)} or {@link EventValueExpression#EventValueExpression(Class)} instead of invoking this
 	 * method repeatedly.
 	 * 
-	 * @param e
-	 * @param c
-	 * @param time
+	 * @param e event

```

Comment:
When adding missing Javadoc, please follow code conventions with the formatting.